File: BowlingGame/ObjC/BowlingGame/Roll.m
//
//  Roll.m
//  BowlingGame
//
//  Created by Tony Hoang on 4/16/18.
//  Copyright © 2018 me. All rights reserved.
//

#import "Roll.h"

@implementation Roll

- (instancetype)initWithScore:(NSInteger)score {
    if (self = [super init]) {
        _score = score;
    }
    
    return self;
}

@end

File: BowlingGame/Swift/BowlingGame/Roll.swift
//
//  Roll.swift
//  BowlingGame
//
//  Created by Tony Hoang on 4/16/18.
//  Copyright © 2018 me. All rights reserved.
//

import Foundation

class Roll: NSObject {
    var score: Int
    
    init(score: Int) {
        self.score = score
        
        super.init()
    }
}
